examples/libf3d/cpp/multi-geom/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ endif()
 # Simple testing
 if(BUILD_TESTING)
   enable_testing()
-  add_test(NAME test_multi-geom COMMAND "$<TARGET_FILE:multi-geom>" "${CMAKE_CURRENT_SOURCE_DIR}/data/")
+  add_test(NAME test_multi-geom COMMAND "$<TARGET_FILE:multi-geom>" "${CMAKE_CURRENT_SOURCE_DIR}/../../data/datasets")
   set_tests_properties(test_multi-geom PROPERTIES
-    PASS_REGULAR_EXPRESSION "Scene bounding box: -0.487464,1,-0.487464,1,-0.5,1")
+    PASS_REGULAR_EXPRESSION "Scene bounding box: -4.44584,5.99809,-3.63704,2.75972,-1.70141,1.70141")
```

Comment:
why would the bounding box change ?

Reply:
Because the data folder change it embed the cow.vtp now.